.travis.yml

```diff
@@ -8,7 +8,7 @@ script:
 - echo "TRAVIS_TAG" $TRAVIS_TAG
 - echo "repo" $TRAVIS_REPO_SLUG
 - if [[ $TRAVIS_PULL_REQUEST != "false" || $TRAVIS_REPO_SLUG != "gearpump/gearpump" ]];
-    then sbt clean coverage +test;
+    then sbt -jvm-opts travis/jvmopts clean coverage +test;
```

Comment:
As it's a concrete build environment profile, better move it to `project/travis/jvmopts`.